deploy/ai-platform/.env.example

```
@@ -10,7 +10,7 @@ DATABASE_POOL_CLOSE_TIMEOUT_SECONDS=5
 DATASTORE_READINESS_TIMEOUT_SECONDS=2
 
 REDIS_PORT=63799
-REDIS_MAX_CONNECTIONS=10
+REDIS_MAX_CONNECTIONS=64
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REDIS_MAX_CONNECTIONS key should go before the REDIS_PORT key

(UnorderedKey)